app/views/content/subjects/engineers-teach-physics/_article.html.erb

```diff
@@ -0,0 +1,120 @@
+
+<article>
+  <section class="statement">
+    <div>
+      <span class="pink xlarge bold">Physics initial teacher training courses for</span>
+    </div>
+
+    <div>
+      <span class="pink medium bold">
+        <ul class="slashed-list">
+          <li>engineers</li>
+          <li>material scientists</li>
+        </ul>
+      </span>
+    </div>
+  </section>
+
+  <section>
+    <h2>The classroom needs you</h2>
+
+    <p>
+      Continue your engineering career by getting into the classroom and use your skills and passion to inspire the next generation.
+    </p>
+
+    <p>
+      We need more physics teachers. Without them children are missing out on the opportunity to develop their understanding of the world around them. Engineers bring a unique and innovative perspective into the physics classroom, helping to explain big things to young minds.
+    </p>
+
+    <p>
+      We want to train a diverse range of physics teachers who can pass on their knowledge in exciting and interesting ways to inspire the engineers of the future.
+    </p>
+  </section>
+
+  <%= render SubjectSpecific::TextBlockWithTwoImagesComponent.new(
+    heading: "Engineering the future",
+    colour: "pink",
+    image_1_path: "media/images/content/hero-images/0017.jpg",
+    image_1_alt: "A teacher sitting with a student helping them with a maths problem",
+    image_2_path: "media/images/content/hero-images/0004.jpg",
+    image_2_alt: "A maths teacher standing at the front of a full classroom delivering a lesson",
+  ) do %>
+
+    <p>
+      Engineers and physicists help find solutions to some of our biggest problems. From climate change to the energy crisis, we need their problem-solving skills and creative thinking to help us survive some of the most critical crises humanity has ever faced.
+    </p>
+
+    <p>
+      The more great physics teachers we have in the classroom today, the more children will grow up inspired to become physicists, engineers and physics teachers tomorrow.
+    </p>
+  <% end %>
+
+  <section>
+    <h2>Physics teacher training programme</h2>
+
+    <p>This new teacher training programme helps engineers become physics teachers. Applications will open at the end of March 2022. To get more information about the course and application process, you can talk to a teacher training adviser. They can help you to:</p>
```

Comment:
is it worth putting in dummy bullets (Provider 1, Provider 2 - rather than the real names, given that it is not confirmed yet) so that you can see how to balance the provider links with the TTA CTA?

Reply:
I think we'll have to redesign that bit when we've got them - I don't think it will be a case of just dropping them into this.